protocol/avdl/keybase1/identify3_ui.avdl

```diff
@@ -0,0 +1,63 @@
+@namespace("keybase.1")
+
+protocol identify3Ui {
+
+  import idl "identify3_common.avdl";
+
+  // showTracker is called when Go wants to force the frontend to show a tracker popup
+  // (rather than the react app triggering the action). In this case, the Go side comes
+  // up with a unique Identify3GUIID.
+  void showTracker(
+    Identify3GUIID guiID,         // id used to tie these things together
+    Identify3Assertion assertion, // username or bob@twitter etc
+    string reason
+  ) oneway;
+
+  enum Identify3RowState {
+    CHECKING_1,
+    VALID_2,
+    ERROR_3,
+    WARNING_4,
+    REVOKED_5
+  }
+
+  enum Identify3RowColor {
+    BLUE_1,
+    RED_2,
+    BLACK_3,
+    GREEN_4
+  }
+
+  @typedef("string")
+  record Identify3RowMeta {}
+
+  @lint("ignore")
+  void identify3UpdateRow(
+    Identify3Assertion guiID,      // id used to tie these things together
```

Comment:
two `guiID`s weird how about assertionID

Reply:
whoops, that just a bug. sorry about that, fixing.